pom.xml

```diff
@@ -244,7 +245,17 @@
             <dependency>
               <groupId>org.apache.curator</groupId>
               <artifactId>curator-test</artifactId>
-              <version>${curator.test.version}</version>
+              <version>${curator.version}</version>
```

Comment:
Why do we need the exclusions in dependency for `maven-javadoc-plugin`?
https://github.com/apache/drill/blob/master/pom.xml#L211
The exclusions for this dependency in dependencyManagement block will take an affect to this too.

Reply:
Removed exclusions block and version.